docs/ai/quickstarts/ai-templates.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Quickstart - Create a .NET AI app using the AI app template
+description: Create a .NET AI app to chat with custom data using the AI app template extensions and the Microsoft.Extensions.AI libraries
+ms.date: 2/21/2025
+ms.topic: quickstart
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: alexwolfmsft
+ms.author: alexwolf
+zone_pivot_groups: meai-targets
+---
+
+# Create a .NET AI app to chat with custom data using the AI app template extensions
```

Comment:
The template also supports switching from a local vector store to Azure AI Search. How do you think we should add that into this document? That would introduce a second pivot that's independent of the first one for AI Model Provider.